mapper/telemetry.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mapper
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	mappingsCount = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "statsd_exporter_loaded_mappings",
```

Comment:
i am not sure what to do about this. I could change the metric name to be package- not exporter-specific, or change the constructor to get it injected from the main package.

Reply:
I'm thinking return the value during load, or keep track of it as a regular value internally.  The caller can then fetch the count and construct the metric however it wants.  This is probably cleaner than creating a new metric in what is effectively a config and parsing package.

In the statsd_exporter case, the prometheus metric would move into main/telemetry and you'd update it whenever the file is loaded.

Reply:
Hmm I don't like the scrape-time plumbing that this implies in every user of the package, and that would need to update with every change of telemetry. The package also already knows about Prometheus and imports the client library, so I think we can let main just pass in the Gauge. That's very flexible for users of the package, but also simple if you just want the damn metrics to work.

Reply:
Makes sense to me 👍